missioncontrol/api/views.py

```diff
@@ -37,6 +43,50 @@ def aggregates(request):
 
 
 def channel_platform_summary(request):
+    '''
+    Lists measures available for specified channel/platform combinations
```

Comment:
To avoid maintenance headache (updating docs in 2 places) I wonder if it might be better to reference the docs/API.md file here instead?

Reply:
Hmm, yeah, we probably at the least shouldn't write the same thing twice, now that I think about it.

I'll remove most of the documentation in the code and reference the API.md file.

As I mentioned in the commit, we may want to move to a more sophisticated documentation solution at some point.